src/main.py

```diff
@@ -87,14 +87,22 @@ def setUp(self):
         # make sure we show the UI before run completes, as otherwise
         # it is possible the run can complete before the UI is shown,
         # causing the app to shut down early
-        self.webview.show()
+        self.view.show()
         return 0
 
     def load_complete(self):
         """
-        This is a PyEverywhere delegate method to let us know the WebView is ready to use. Just pass for now.
+        This is a PyEverywhere delegate method to let us know the WebView is ready to use.
         """
-        pass
+
+        # On Android, there is a system back button, that works like the browser back button. Make sure we clear the
+        # history after first load so that the user cannot go back to the loading screen. We cannot clear the history
+        # during load, so we do it here.
+        # For more info, see: https://stackoverflow.com/questions/8103532/how-to-clear-webview-history-in-android
+        if pew.ui.platform == 'android':
```

Comment:
Is there a reason you're adding this so regularly? Is it so that you can test locally?

I've been trying to clean this up, operating under the assumption that this file will _always_ run under android.

Reply:
Yes, this is for local testing. Plus, I have a feeling people will be asking for app versions of Kolibri on other platforms as well, so I figured it doesn't hurt to have these lines in there. (I actually tried to use it to test some iOS issues since Safari is blocked, but I stopped with a dependency packaging issue.)